physics/drag_suite.F90

```diff
@@ -523,6 +539,12 @@ subroutine drag_suite_run(                                           &
       Tdtend = dtidx(index_of_temperature,index_of_process_orographic_gwd)
    endif
 
+
+   ! Initialize winds for sequential updating
+   ! NOTE: These will only be updated if ugwp_seq_update == .true.
+   uwnd1(:,:) = u1(:,:)
```

Comment:
Extremely minor point, but if these variables (`u/vwnd1`) are only ever used when `ugwp_seq_update` is T, it would save a bit of time to not initialize them here for the (default?) case when `ugwp_seq_update' is F.

Reply:
The variables are only updated if ugwp_seq_update is T, but they are still used, even if it is F.  "u/vwnd1" substitute for u1 and v1 further down in the code.